includes/abstracts/abstract-wc-data.php

```diff
@@ -398,7 +398,7 @@ public function add_meta_data( $key, $value, $unique = false ) {
 	 *
 	 * @param  string $key
 	 * @param  string $value
-	 * @param  int $meta_id
+	 * @param  string $meta_id
```

Comment:
Should be int.

Reply:
OK. Then we should change `$meta_id = ''` to `$meta_id = 0` at line *403*, cause empty string and 0 are same for `if` condition check.